Add verification for statefulset (#307)
submariner-io/submariner-website: src/content/quickstart/verify_with_discovery.md

curl nginx.default.svc.clusterset.local:8080
```

> To access service in a specific cluster, prefix query with `<cluster-id>` as follows:

Reviewer: ```suggestion
To access a Service in a specific cluster, prefix the query with `<cluster-id>` as follows:
```

```bash
curl cluster-a.nginx.default.svc.clusterset.local:8080
```

#### Verify StatefulSets

> StatefulSets use headless Service, so only work with non-globalnet deployments. Create a `web.yaml` as follows:

Reviewer: I don't think the reference to globalnet here is relevant.

```suggestion
A StatefulSet uses a headless Service. Create a `web.yaml` as follows:
```

```yaml
apiVersion: v1
kind: Service
metadata:
  name: nginx-ss
  labels:
    app.kubernetes.io/instance: nginx-ss
    app.kubernetes.io/name: nginx-ss
spec:
  ports:
  - port: 80
    name: web
  clusterIP: None
  selector:
    app.kubernetes.io/instance: nginx-ss
    app.kubernetes.io/name: nginx-ss
---
apiVersion: apps/v1
kind: StatefulSet
metadata:
  name: web
spec:
  serviceName: "nginx-ss"
  replicas: 2
  selector:
    matchLabels:
        app.kubernetes.io/instance: nginx-ss
        app.kubernetes.io/name: nginx-ss
  template:
    metadata:
      labels:
        app.kubernetes.io/instance: nginx-ss
        app.kubernetes.io/name: nginx-ss
    spec:
      containers:
      - name: nginx-ss
        image: nginxinc/nginx-unprivileged:stable-alpine
        ports:
        - containerPort: 80
          name: web
```

> Use this yaml to create a StatefulSet `web` with `nginx-ss` as headless service.

Reviewer: ```suggestion
Use this yaml to create a StatefulSet `web` with `nginx-ss` as the headless Service.
```

```bash
export KUBECONFIG=cluster-a/auth/kubeconfig
kubectl -n default  apply -f web.yaml
curl nginx-ss.default.svc.clusterset.local:8080
```

> To access service in a specific cluster, prefix query with `<cluster-id>` as follows:

Reviewer: ```suggestion
To access the Service in a specific cluster, prefix the query with `<cluster-id>`:
```

```bash
curl cluster-a.nginx-ss.default.svc.clusterset.local:8080
```

> To access individual pod in a specific cluster, prefix query with `<pod-hostname>.<cluster-id>` as follows:

Reviewer: ```suggestion
To access an individual pod in a specific cluster, prefix the query with `<pod-hostname>.<cluster-id>`:
```
